src/Files.App/Helpers/FileExtensionHelpers.cs

```diff
@@ -11,12 +11,10 @@ public static class FileExtensionHelpers
         /// <param name="fileExtensionToCheck">The file extension to check.</param>
         /// <returns><c>true</c> if the fileExtensionToCheck is an image;
         /// otherwise, <c>false</c>.</returns>
-        public static bool IsImageFile(string fileExtensionToCheck)
+        public static bool IsImageFile(string? fileExtensionToCheck)
         {
-            if (string.IsNullOrEmpty(fileExtensionToCheck))
-            {
+            if (string.IsNullOrWhiteSpace(fileExtensionToCheck))
```

Comment:
`if` block needs to have the curly braces I believe.

Reply:
![1](https://user-images.githubusercontent.com/102259289/196406318-f609cec0-e383-4d3f-9944-f1cfe5731d08.png)

Reply:
My bad!